packages/typescriptlang-org/src/components/search/ResultRow.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useState } from "react"
+import { FormattedRelativeTime } from "react-intl"
+
+import { cx } from "../../lib/cx"
+import { SearchHit } from "./types"
+
+import "./ResultRow.scss"
+
+export type ResultRowprops = {
+  exactMatch?: boolean
+  installer: [string, string]
+  hit: SearchHit
+}
+
+export const ResultRow: React.FC<ResultRowprops> = ({
+  exactMatch,
+  installer,
+  hit: {
+    description,
+    downloadsLast30Days,
+    humanDownloadsLast30Days,
+    modified,
+    name,
+    repository,
+    types,
+  },
+}) => {
+  const npmUrl = `https://www.npmjs.com/package/${name}`
+  const [icon, label, viaUrl] =
+    types.ts === "included"
+      ? ["in", "included", repository?.url || npmUrl]
+      : [
+          "dt",
+          "from Definitely Typed",
+          `https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/${name}`,
+        ]
+
+  const initialCopyStatus = ["copy", `Copy ${name} installation script`]
```

Comment:
Question: is there a better icon set we should use instead of `"copy"`/`"copied"`/`":("`? E.g. https://developer.microsoft.com/en-us/fluentui#/styles/web/icons, but just SVGs?